docs/source/allen.rst

```diff
@@ -70,20 +70,21 @@ Single image download
 
 Full-blown registration
 ~~~~~~~~~~~~~~~~~~~~~~~
-The most useful function is the :code:`download_dataset`. It expects the user to specify the unique
-:code:`dataset_id`. It returns a generator. Each iteration then yields
+The most useful function is the :code:`download_dataset_parallel`.
+It expects the user to specify the unique :code:`dataset_id`. It returns a
```

Comment:
Just a suggestion, what do you think about explaining why we call this `parallel` ? I just checked and it is in the docstring but maybe we can be more explicit about it. However, I don't mind keeping it that way!

Reply:
Good point:) I will add a short description. Note that I am also planning to dedicate a new PR to documentation of this functionality #33

Reply:
Added some text here e6af21ba872778c52788356df1415d1e27bf254f